make/test.mk

```diff
@@ -25,7 +25,7 @@ test-with-coverage:
 	@echo "running the tests with coverage..."
 	@-mkdir -p $(COV_DIR)
 	@-rm $(COV_DIR)/coverage.txt
-	$(Q)go test -vet off ${V_FLAG} $(shell go list ./... | grep -v /test/e2e) -coverprofile=$(COV_DIR)/coverage.txt -covermode=atomic ./...
+	$(Q)go test -vet off ${V_FLAG} $(shell go list ./... | grep -v /cmd/manager) -coverprofile=$(COV_DIR)/coverage.txt -covermode=atomic ./...
```

Comment:
doesn't that mean that it won't run the tests inside `cmd/manager` directory? We have one small test there already: https://github.com/codeready-toolchain/member-operator/blob/master/cmd/manager/main_test.go